upcloud/resource_upcloud_server.go

```diff
@@ -712,11 +711,11 @@ func buildNetworkOpts(d *schema.ResourceData, meta interface{}) ([]request.Creat
 			Type: d.Get(keyRoot + "type").(string),
 		}
 
-		if v, ok := d.GetOk(keyRoot + "source_ip_filtering"); ok {
+		if v, ok := d.GetOkExists(keyRoot + "source_ip_filtering"); ok {
```

Comment:
>// Deprecated: usage is discouraged due to undefined behaviors and may be
>// removed in a future version of the SDK

This is when I looked into what this does.
Hmmm. Though they have changed the comment now to read. When I looked at the github repo.

>// This should only be used if absolutely required/needed.

There are more in the comment, but I guess this could be used here.

Reply:
updated to use .Get instead. Also replaced config in the schema to have default values